trestle/core/commands/split.py

```diff
@@ -129,7 +201,9 @@ def split_model_at_path_chain(
         cur_path_index: int,
         split_plan: Plan,
         strip_root: bool,
-        root_file_name: str = ''
+        root_file_name: str,
+        aliases_to_strip: Dict[str, AliasTracker],
+        last_one: bool = True
```

Comment:
If you have the time would be great to structure docstrings.

Reply:
Done